src/random.h

```diff
@@ -67,9 +67,21 @@
  * Thread-safe.
  */
 void GetRandBytes(unsigned char* buf, int num) noexcept;
+/** Generate a random integer in the range [0..range). Precondition: range > 0 */
```

Comment:
When you're changing these comments anyway, you might want to add that the random number comes from a uniform distribution (especially for durations this is not obvious).

Reply:
Thanks, fixed